Command/DoctrineCommandHelper.php

```diff
@@ -56,11 +56,14 @@ public static function setApplicationConnection(Application $application, $sessi
      * @param Application $application
      * @param string      $sessionName
      */
-    public static function setApplicationPHPCRSession(Application $application, $sessionName)
+    public static function setApplicationPHPCRSession(Application $application, $sessionName, $admin = false)
     {
-        /** @var $registry ManagerRegistry */
-        $registry = $application->getKernel()->getContainer()->get('doctrine_phpcr');
-        $session = $registry->getConnection($sessionName);
```

Comment:
we could add a getAdminForConnection($sessionName) to the registry to move the logic into the registry. would that make sense?